src/react-components/src/components/03-organisms/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,50 @@
+
```

Comment:
Newline

I've conceptually been grouping Accordions, Modals, and Tab Sets together in something I've called "Content Displays". The idea is that they all hide/show content. We're not really using the atom/molecule/organism structure within Storybook; might be worth discussing if/when/how we might deprecate that model of organization in favor of something else.

[Figma docs for content displays](https://www.figma.com/file/qShodlfNCJHb8n03IFyApM/Master?node-id=10734%3A2520)

Subpages for other design groupings:
![Screen Shot 2020-04-10 at 10 16 37 AM](https://user-images.githubusercontent.com/1720093/78997333-5e4eb400-7b14-11ea-93f3-da80275a5c0c.png)

Reply:
The arrangement of stuff within folders is kind of messy and will touch everything, and cause breaking changes.  I'm thinking this is a good stand-alone task and we should probably do it all at once.